### PR TITLE
feat: Few more German translations 🇩🇪

### PR DESCRIPTION
**Pull request by ParkerEde**
Summary of changes:
I've made a few more german translations and fix one too long translation for QX7

### CODE REVIEW

File: radio/src/translations/de.h [hunk 1]
```diff
@@ -39,18 +39,18 @@
 #define TR_VBLMODE                     "AUS","Taste","Stks","Beide","EIN"
 #define TR_TRNMODE                     "AUS","+=",":="
 #define TR_TRNCHN                      "CH1","CH2","CH3","CH4"
-#define TR_AUX_SERIAL_MODES            "AUS","Telem Mirror","Telemetry In","SBUS Eingang","LUA","CLI","GPS","Debug","SpaceMouse"
+#define TR_AUX_SERIAL_MODES            "AUS","Telem Spiegel","Telemetrie In","SBUS Eingang","LUA","CLI","GPS","Debug","SpaceMouse"
```
Comment (froqstar) [inline radio/src/translations/de.h:42]: "Spiegel" is a bit too literal IMO, I would not get the meaning as a newcomer.
How about "weiterl."?

Concerning the telemetry in, AFAIK it currently only supports FrSky Hub telemetry in, so could we just call it "FrSky Hub In" until other protocols are supported?

File: radio/src/translations/de.h [hunk 2]
```diff
@@ -342,16 +342,16 @@
...
+#define TR_EXPONAME                    TR("Name", "Zeilenname")
 #define TR_BITMAP                      "Modellfoto"
 #define TR_NO_PICTURE                  "kein Foto"
 #define TR_TIMER                       "Timer"
 #define TR_START                       "Start"
 #define TR_ELIMITS                     TR("Erw. Limit", "Erw. Wege auf 150%")
 #define TR_ETRIMS                      TR("Erw. Trims", "Erw. Trim  auf 100%")
 #define TR_TRIMINC                     TR("Trimschritt", "Trimmschritte")
-#define TR_DISPLAY_TRIMS               TR("Trimanzeige", "Trimwerte anzeigen")
+#define TR_DISPLAY_TRIMS               TR("Trimanzeige", "Trimmwerte anzeigen")
```
Comment (froqstar) [inline radio/src/translations/de.h:354]: Does it need to be changed to "Trimmanzeige" then as well?

File: radio/src/translations/de.h [hunk 3]
```diff
@@ -888,29 +888,29 @@
...
+#define TR_ANADIAGS_UNFILTRAW          "analoge Geber ungefiltert und unbearbeitet"
+#define TR_ANADIAGS_MINMAX             "Min., Max. und Bereich"
 #define TR_ANADIAGS_MOVE               "Move analogs to their extremes!"
 #define TR_SPEAKER                     INDENT "Lautspr"
 #define TR_BUZZER                      INDENT "Summer"
 #define TR_BYTES                       "Bytes"
-#define TR_MODULE_BIND                 BUTTON(TR("Bnd","Bind"))   //9XR-Pro
-#define TR_POWERMETER_ATTN_NEEDED      "Attenuator needed"
-#define TR_PXX2_SELECT_RX              "Select RX"
+#define TR_MODULE_BIND                 BUTTON(TR("Bnd","Binde"))   //9XR-Pro
```
Comment (froqstar) [inline radio/src/translations/de.h:899]: Is this the label of the "bind" button? If so, I would prefer "Binden", "Binde" sounds more of a status than an action.

File: radio/src/translations/de.h [hunk 4]
```diff
@@ -1152,8 +1152,8 @@
 #define TR_MENU_INVERT                 "Invertieren<!>"
 #define TR_JITTER_FILTER               "ADC Filter"
 #define TR_DEAD_ZONE                   "Dead zone"
-#define TR_RTC_CHECK                   TR("Check RTC", "Check RTC voltage")
-#define TR_AUTH_FAILURE                "Auth-failure"
+#define TR_RTC_CHECK                   TR("Prüfe RTC", "Prüfe RTC Spannung")
```
Comment (froqstar) [inline radio/src/translations/de.h:1155]: If this is a warning to prompt the user to check the RTC battery, this sounds like it is currently checking the voltage.
"RTC Prüfen" and "RTC Spannung prüfen"?

**Review by froqstar — COMMENTED** [review]
Not sure yet about all the "Suche" vs. "Suchen" and "Binde" vs. "Binden" yet. I guess I have to hold it in my hands and try it to form a proper opinion. And as always, personal taste.

Thanks for your work :)

File: radio/src/translations/de.h (hunk 1 above)
Comment (ParkerEde) [reply radio/src/translations/de.h:42]: yes, you are right. I will change to "weiterl." It's a good Idea I think.

File: radio/src/translations/de.h (hunk 2 above)
Comment (ParkerEde) [reply radio/src/translations/de.h:354]: done

File: radio/src/translations/de.h (hunk 3 above)
Comment (ParkerEde) [reply radio/src/translations/de.h:899]: Yes it's the Bind Button.
done

File: radio/src/translations/de.h (hunk 4 above)
Comment (ParkerEde) [reply radio/src/translations/de.h:1155]: done

Comment (ParkerEde) [reply radio/src/translations/de.h:1155]: with "RTC Spannung prüfen" there is a line break.
image

Should I then call it "RTC Volt prüfen"?

Comment (ParkerEde) [reply radio/src/translations/de.h:1155]: image

**Review by froqstar — APPROVED** [review]

File: radio/src/translations/de.h (hunk 4 above)
Comment (ParkerEde) [reply radio/src/translations/de.h]: new plan: "Akku Spannungsbereich" is also much too long. So I've shorten it with the same abbreviation like "RTC Spannung prüfen". I think it looks good:
image

**Comment by pfeerick** [comment]
Just say when you're ready ;)

**Comment by ParkerEde** [comment]
@pfeerick 
Question. On X10 in SYS/Hardware under the item "Test" the two buttons "Analogs" and "Switches" are not translatable. Is this correct or have I overseen something?

**Comment by pfeerick** [comment]
Those *should* be:
- STR_ANALOGS_BTN => TR_ANALOGS_BTN
- STR_KEYS_BTN => TR_KEYS_BTN

**Comment by ParkerEde** [comment]
@pfeerick 
oh my god, yes thank you :)
now I'm ready...

**Review by pfeerick — APPROVED** [review]

**Pull request merged**